19:17:08 DEBUG : Logs will be stored at '/home/rosario/Documents/CR/cr-hardware-acceleration/IDE.log'.
19:17:11 INFO  : Platform repository initialization has completed.
19:17:11 INFO  : Launching XSCT server: xsct -n  -interactive /home/rosario/Documents/CR/cr-hardware-acceleration/temp_xsdb_launch_script.tcl
19:17:11 ERROR : (XSDB Server)which: no xlsclients in (/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_be/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.21.4/bin::/tools/Xilinx/Vivado/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vivado/2022.2/ids_lite/ISE/bin/lin64:/tools/Xilinx/Vitis/2022.2/aietools/bin:/tools/Xilinx/Vivado/2022.2/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib/python3.8/site-packages::/tools/Xilinx/Vivado/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vitis_HLS/2022.2/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.3.2/bin:/tools/Xilinx/DocNav:/home/rosario/.local/bin:/home/rosario/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/var/lib/snapd/snap/bin)

19:17:11 INFO  : Registering command handlers for Vitis TCF services
19:20:11 ERROR : Timed out. 180 seconds have elapsed while waiting for XSCT server to launch.
19:20:11 INFO  : Launching XSCT server: xsct -n  -interactive /home/rosario/Documents/CR/cr-hardware-acceleration/temp_xsdb_launch_script.tcl
19:20:11 ERROR : Failed to call init_repo
Reason: xsct server communication channel is not established.
19:20:11 ERROR : Exception raised in executing the command =Failed to call init_repo
Reason: xsct server communication channel is not established. 
19:20:11 ERROR : (XSDB Server)which: no xlsclients in (/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_be/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.21.4/bin::/tools/Xilinx/Vivado/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vivado/2022.2/ids_lite/ISE/bin/lin64:/tools/Xilinx/Vitis/2022.2/aietools/bin:/tools/Xilinx/Vivado/2022.2/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib/python3.8/site-packages::/tools/Xilinx/Vivado/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vitis_HLS/2022.2/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.3.2/bin:/tools/Xilinx/DocNav:/home/rosario/.local/bin:/home/rosario/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/var/lib/snapd/snap/bin)

19:23:11 ERROR : Timed out. 180 seconds have elapsed while waiting for XSCT server to launch.
19:23:11 ERROR : xsct server communication channel is not established.
19:23:11 INFO  : Launching XSCT server: xsct -n  -interactive /home/rosario/Documents/CR/cr-hardware-acceleration/temp_xsdb_launch_script.tcl
19:23:11 ERROR : (XSDB Server)which: no xlsclients in (/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_be/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.21.4/bin::/tools/Xilinx/Vivado/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vivado/2022.2/ids_lite/ISE/bin/lin64:/tools/Xilinx/Vitis/2022.2/aietools/bin:/tools/Xilinx/Vivado/2022.2/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib/python3.8/site-packages::/tools/Xilinx/Vivado/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vitis_HLS/2022.2/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.3.2/bin:/tools/Xilinx/DocNav:/home/rosario/.local/bin:/home/rosario/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/var/lib/snapd/snap/bin)

19:26:11 ERROR : Timed out. 180 seconds have elapsed while waiting for XSCT server to launch.
19:26:11 INFO  : Launching XSCT server: xsct -n  -interactive /home/rosario/Documents/CR/cr-hardware-acceleration/temp_xsdb_launch_script.tcl
19:26:11 ERROR : (XSDB Server)which: no xlsclients in (/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_be/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.21.4/bin::/tools/Xilinx/Vivado/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vivado/2022.2/ids_lite/ISE/bin/lin64:/tools/Xilinx/Vitis/2022.2/aietools/bin:/tools/Xilinx/Vivado/2022.2/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib/python3.8/site-packages::/tools/Xilinx/Vivado/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vitis_HLS/2022.2/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.3.2/bin:/tools/Xilinx/DocNav:/home/rosario/.local/bin:/home/rosario/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/var/lib/snapd/snap/bin)

19:29:11 ERROR : Timed out. 180 seconds have elapsed while waiting for XSCT server to launch.
19:29:11 ERROR : Failed to query RDI_DATADIR
Reason: xsct server communication channel is not established.
19:29:11 ERROR : Exception raised in executing the command =Failed to query RDI_DATADIR
Reason: xsct server communication channel is not established. 
19:35:42 DEBUG : Logs will be stored at '/home/rosario/Documents/CR/cr-hardware-acceleration/IDE.log'.
19:35:43 INFO  : Launching XSCT server: xsct -n  -interactive /home/rosario/Documents/CR/cr-hardware-acceleration/temp_xsdb_launch_script.tcl
19:35:43 ERROR : (XSDB Server)which: no xlsclients in (/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_be/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.21.4/bin::/tools/Xilinx/Vivado/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vivado/2022.2/ids_lite/ISE/bin/lin64:/tools/Xilinx/Vitis/2022.2/aietools/bin:/tools/Xilinx/Vivado/2022.2/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib/python3.8/site-packages::/tools/Xilinx/Vivado/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vitis_HLS/2022.2/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.3.2/bin:/tools/Xilinx/DocNav:/home/rosario/.local/bin:/home/rosario/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/var/lib/snapd/snap/bin)

19:35:43 INFO  : Platform repository initialization has completed.
19:35:43 INFO  : Registering command handlers for Vitis TCF services
19:38:43 ERROR : Timed out. 180 seconds have elapsed while waiting for XSCT server to launch.
19:38:43 INFO  : Launching XSCT server: xsct -n  -interactive /home/rosario/Documents/CR/cr-hardware-acceleration/temp_xsdb_launch_script.tcl
19:38:43 ERROR : Failed to call init_repo
Reason: xsct server communication channel is not established.
19:38:43 ERROR : Exception raised in executing the command =Failed to call init_repo
Reason: xsct server communication channel is not established. 
19:38:43 ERROR : (XSDB Server)which: no xlsclients in (/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_be/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.21.4/bin::/tools/Xilinx/Vivado/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vivado/2022.2/ids_lite/ISE/bin/lin64:/tools/Xilinx/Vitis/2022.2/aietools/bin:/tools/Xilinx/Vivado/2022.2/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib/python3.8/site-packages::/tools/Xilinx/Vivado/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vitis_HLS/2022.2/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.3.2/bin:/tools/Xilinx/DocNav:/home/rosario/.local/bin:/home/rosario/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/var/lib/snapd/snap/bin)

19:41:43 ERROR : Timed out. 180 seconds have elapsed while waiting for XSCT server to launch.
19:41:43 INFO  : Launching XSCT server: xsct -n  -interactive /home/rosario/Documents/CR/cr-hardware-acceleration/temp_xsdb_launch_script.tcl
19:41:43 ERROR : xsct server communication channel is not established.
19:41:43 ERROR : (XSDB Server)which: no xlsclients in (/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_be/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.21.4/bin::/tools/Xilinx/Vivado/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vivado/2022.2/ids_lite/ISE/bin/lin64:/tools/Xilinx/Vitis/2022.2/aietools/bin:/tools/Xilinx/Vivado/2022.2/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib/python3.8/site-packages::/tools/Xilinx/Vivado/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vitis_HLS/2022.2/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.3.2/bin:/tools/Xilinx/DocNav:/home/rosario/.local/bin:/home/rosario/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/var/lib/snapd/snap/bin)

19:44:43 ERROR : Timed out. 180 seconds have elapsed while waiting for XSCT server to launch.
19:44:43 INFO  : Launching XSCT server: xsct -n  -interactive /home/rosario/Documents/CR/cr-hardware-acceleration/temp_xsdb_launch_script.tcl
19:44:43 ERROR : (XSDB Server)which: no xlsclients in (/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_be/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.21.4/bin::/tools/Xilinx/Vivado/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vivado/2022.2/ids_lite/ISE/bin/lin64:/tools/Xilinx/Vitis/2022.2/aietools/bin:/tools/Xilinx/Vivado/2022.2/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib/python3.8/site-packages::/tools/Xilinx/Vivado/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vitis_HLS/2022.2/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.3.2/bin:/tools/Xilinx/DocNav:/home/rosario/.local/bin:/home/rosario/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/var/lib/snapd/snap/bin)

19:47:43 ERROR : Timed out. 180 seconds have elapsed while waiting for XSCT server to launch.
19:47:43 ERROR : Failed to openhw "/tmp/hwspec_mb_design_wrapper2240378294526609815/mb_design_wrapper.xsa"
Reason: xsct server communication channel is not established.
19:47:43 INFO  : Launching XSCT server: xsct -n  -interactive /home/rosario/Documents/CR/cr-hardware-acceleration/temp_xsdb_launch_script.tcl
19:47:43 ERROR : (XSDB Server)which: no xlsclients in (/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_be/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.21.4/bin::/tools/Xilinx/Vivado/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vivado/2022.2/ids_lite/ISE/bin/lin64:/tools/Xilinx/Vitis/2022.2/aietools/bin:/tools/Xilinx/Vivado/2022.2/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib/python3.8/site-packages::/tools/Xilinx/Vivado/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vitis_HLS/2022.2/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.3.2/bin:/tools/Xilinx/DocNav:/home/rosario/.local/bin:/home/rosario/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/var/lib/snapd/snap/bin)

19:50:43 ERROR : Timed out. 180 seconds have elapsed while waiting for XSCT server to launch.
19:50:43 ERROR : Failed to query RDI_DATADIR
Reason: xsct server communication channel is not established.
19:50:43 INFO  : Launching XSCT server: xsct -n  -interactive /home/rosario/Documents/CR/cr-hardware-acceleration/temp_xsdb_launch_script.tcl
19:50:43 ERROR : Exception raised in executing the command =Failed to query RDI_DATADIR
Reason: xsct server communication channel is not established. 
19:50:43 ERROR : (XSDB Server)which: no xlsclients in (/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_be/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.21.4/bin::/tools/Xilinx/Vivado/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vivado/2022.2/ids_lite/ISE/bin/lin64:/tools/Xilinx/Vitis/2022.2/aietools/bin:/tools/Xilinx/Vivado/2022.2/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib/python3.8/site-packages::/tools/Xilinx/Vivado/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vitis_HLS/2022.2/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.3.2/bin:/tools/Xilinx/DocNav:/home/rosario/.local/bin:/home/rosario/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/var/lib/snapd/snap/bin)

19:53:43 ERROR : Timed out. 180 seconds have elapsed while waiting for XSCT server to launch.
19:53:43 ERROR : Failed to openhw "/tmp/hwspec_mb_design_wrapper2240378294526609815/mb_design_wrapper.xsa"
Reason: xsct server communication channel is not established.
19:53:43 INFO  : Launching XSCT server: xsct -n  -interactive /home/rosario/Documents/CR/cr-hardware-acceleration/temp_xsdb_launch_script.tcl
19:53:43 ERROR : (XSDB Server)which: no xlsclients in (/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_be/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.21.4/bin::/tools/Xilinx/Vivado/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vivado/2022.2/ids_lite/ISE/bin/lin64:/tools/Xilinx/Vitis/2022.2/aietools/bin:/tools/Xilinx/Vivado/2022.2/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib/python3.8/site-packages::/tools/Xilinx/Vivado/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vitis_HLS/2022.2/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.3.2/bin:/tools/Xilinx/DocNav:/home/rosario/.local/bin:/home/rosario/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/var/lib/snapd/snap/bin)

19:56:43 ERROR : Timed out. 180 seconds have elapsed while waiting for XSCT server to launch.
19:56:43 ERROR : Failed to openhw "/tmp/hwspec_mb_design_wrapper2240378294526609815/mb_design_wrapper.xsa"
Reason: xsct server communication channel is not established.
19:57:58 INFO  : Launching XSCT server: xsct -n  -interactive /home/rosario/Documents/CR/cr-hardware-acceleration/temp_xsdb_launch_script.tcl
19:57:58 ERROR : (XSDB Server)which: no xlsclients in (/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_be/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.21.4/bin::/tools/Xilinx/Vivado/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vivado/2022.2/ids_lite/ISE/bin/lin64:/tools/Xilinx/Vitis/2022.2/aietools/bin:/tools/Xilinx/Vivado/2022.2/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib/python3.8/site-packages::/tools/Xilinx/Vivado/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vitis_HLS/2022.2/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.3.2/bin:/tools/Xilinx/DocNav:/home/rosario/.local/bin:/home/rosario/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/var/lib/snapd/snap/bin)

20:00:58 ERROR : Timed out. 180 seconds have elapsed while waiting for XSCT server to launch.
20:00:58 ERROR : Failed to openhw "/tmp/hwspec_mb_design_wrapper2240378294526609815/mb_design_wrapper.xsa"
Reason: xsct server communication channel is not established.
20:01:23 INFO  : Launching XSCT server: xsct -n  -interactive /home/rosario/Documents/CR/cr-hardware-acceleration/temp_xsdb_launch_script.tcl
20:01:23 ERROR : (XSDB Server)which: no xlsclients in (/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_be/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.21.4/bin::/tools/Xilinx/Vivado/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vivado/2022.2/ids_lite/ISE/bin/lin64:/tools/Xilinx/Vitis/2022.2/aietools/bin:/tools/Xilinx/Vivado/2022.2/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib/python3.8/site-packages::/tools/Xilinx/Vivado/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vitis_HLS/2022.2/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.3.2/bin:/tools/Xilinx/DocNav:/home/rosario/.local/bin:/home/rosario/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/var/lib/snapd/snap/bin)

20:04:23 ERROR : Timed out. 180 seconds have elapsed while waiting for XSCT server to launch.
20:04:23 ERROR : Failed to openhw "/tmp/hwspec_mb_design_wrapper2240378294526609815/mb_design_wrapper.xsa"
Reason: xsct server communication channel is not established.
20:04:23 INFO  : Launching XSCT server: xsct -n  -interactive /home/rosario/Documents/CR/cr-hardware-acceleration/temp_xsdb_launch_script.tcl
20:04:23 ERROR : (XSDB Server)which: no xlsclients in (/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_be/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.21.4/bin::/tools/Xilinx/Vivado/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vivado/2022.2/ids_lite/ISE/bin/lin64:/tools/Xilinx/Vitis/2022.2/aietools/bin:/tools/Xilinx/Vivado/2022.2/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib/python3.8/site-packages::/tools/Xilinx/Vivado/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vitis_HLS/2022.2/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.3.2/bin:/tools/Xilinx/DocNav:/home/rosario/.local/bin:/home/rosario/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/var/lib/snapd/snap/bin)

20:07:23 ERROR : Timed out. 180 seconds have elapsed while waiting for XSCT server to launch.
20:07:23 ERROR : Failed to openhw "/tmp/hwspec_mb_design_wrapper2240378294526609815/mb_design_wrapper.xsa"
Reason: xsct server communication channel is not established.
20:20:48 DEBUG : Logs will be stored at '/home/rosario/Documents/CR/cr-hardware-acceleration/IDE.log'.
20:20:51 INFO  : Launching XSCT server: xsct -n  -interactive /home/rosario/Documents/CR/cr-hardware-acceleration/temp_xsdb_launch_script.tcl
20:20:51 ERROR : (XSDB Server)which: no xlsclients in (/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_be/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.21.4/bin::/tools/Xilinx/Vivado/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vivado/2022.2/ids_lite/ISE/bin/lin64:/tools/Xilinx/Vitis/2022.2/aietools/bin:/tools/Xilinx/Vivado/2022.2/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib/python3.8/site-packages::/tools/Xilinx/Vivado/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vitis_HLS/2022.2/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.3.2/bin:/tools/Xilinx/DocNav:/home/rosario/.local/bin:/home/rosario/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/var/lib/snapd/snap/bin)

20:20:52 INFO  : Registering command handlers for Vitis TCF services
20:20:52 INFO  : Platform repository initialization has completed.
20:23:51 ERROR : Timed out. 180 seconds have elapsed while waiting for XSCT server to launch.
20:23:51 INFO  : Launching XSCT server: xsct -n  -interactive /home/rosario/Documents/CR/cr-hardware-acceleration/temp_xsdb_launch_script.tcl
20:23:51 ERROR : (XSDB Server)which: no xlsclients in (/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_be/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.21.4/bin::/tools/Xilinx/Vivado/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vivado/2022.2/ids_lite/ISE/bin/lin64:/tools/Xilinx/Vitis/2022.2/aietools/bin:/tools/Xilinx/Vivado/2022.2/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib/python3.8/site-packages::/tools/Xilinx/Vivado/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vitis_HLS/2022.2/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.3.2/bin:/tools/Xilinx/DocNav:/home/rosario/.local/bin:/home/rosario/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/var/lib/snapd/snap/bin)

20:26:51 ERROR : Timed out. 180 seconds have elapsed while waiting for XSCT server to launch.
20:26:51 INFO  : Launching XSCT server: xsct -n  -interactive /home/rosario/Documents/CR/cr-hardware-acceleration/temp_xsdb_launch_script.tcl
20:26:51 ERROR : xsct server communication channel is not established.
20:26:51 ERROR : (XSDB Server)which: no xlsclients in (/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_be/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.21.4/bin::/tools/Xilinx/Vivado/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vivado/2022.2/ids_lite/ISE/bin/lin64:/tools/Xilinx/Vitis/2022.2/aietools/bin:/tools/Xilinx/Vivado/2022.2/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib/python3.8/site-packages::/tools/Xilinx/Vivado/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vitis_HLS/2022.2/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.3.2/bin:/tools/Xilinx/DocNav:/home/rosario/.local/bin:/home/rosario/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/var/lib/snapd/snap/bin)

20:29:51 ERROR : Timed out. 180 seconds have elapsed while waiting for XSCT server to launch.
20:29:51 INFO  : Launching XSCT server: xsct -n  -interactive /home/rosario/Documents/CR/cr-hardware-acceleration/temp_xsdb_launch_script.tcl
20:29:51 ERROR : Failed to call init_repo
Reason: xsct server communication channel is not established.
20:29:51 ERROR : Exception raised in executing the command =Failed to call init_repo
Reason: xsct server communication channel is not established. 
20:29:51 ERROR : (XSDB Server)which: no xlsclients in (/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_be/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.21.4/bin::/tools/Xilinx/Vivado/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vivado/2022.2/ids_lite/ISE/bin/lin64:/tools/Xilinx/Vitis/2022.2/aietools/bin:/tools/Xilinx/Vivado/2022.2/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib/python3.8/site-packages::/tools/Xilinx/Vivado/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vitis_HLS/2022.2/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.3.2/bin:/tools/Xilinx/DocNav:/home/rosario/.local/bin:/home/rosario/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/var/lib/snapd/snap/bin)

20:32:51 ERROR : Timed out. 180 seconds have elapsed while waiting for XSCT server to launch.
20:32:51 INFO  : Launching XSCT server: xsct -n  -interactive /home/rosario/Documents/CR/cr-hardware-acceleration/temp_xsdb_launch_script.tcl
20:32:51 ERROR : Failed to openhw "/tmp/hwspec_mb_design_wrapper9216242644888787270/mb_design_wrapper.xsa"
Reason: xsct server communication channel is not established.
20:32:51 ERROR : (XSDB Server)which: no xlsclients in (/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_be/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.21.4/bin::/tools/Xilinx/Vivado/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vivado/2022.2/ids_lite/ISE/bin/lin64:/tools/Xilinx/Vitis/2022.2/aietools/bin:/tools/Xilinx/Vivado/2022.2/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib/python3.8/site-packages::/tools/Xilinx/Vivado/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vitis_HLS/2022.2/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.3.2/bin:/tools/Xilinx/DocNav:/home/rosario/.local/bin:/home/rosario/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/var/lib/snapd/snap/bin)

20:35:51 ERROR : Timed out. 180 seconds have elapsed while waiting for XSCT server to launch.
20:35:51 ERROR : Failed to query RDI_DATADIR
Reason: xsct server communication channel is not established.
20:35:51 INFO  : Launching XSCT server: xsct -n  -interactive /home/rosario/Documents/CR/cr-hardware-acceleration/temp_xsdb_launch_script.tcl
20:35:51 ERROR : Exception raised in executing the command =Failed to query RDI_DATADIR
Reason: xsct server communication channel is not established. 
20:35:51 ERROR : (XSDB Server)which: no xlsclients in (/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_be/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.21.4/bin::/tools/Xilinx/Vivado/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vivado/2022.2/ids_lite/ISE/bin/lin64:/tools/Xilinx/Vitis/2022.2/aietools/bin:/tools/Xilinx/Vivado/2022.2/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib/python3.8/site-packages::/tools/Xilinx/Vivado/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vitis_HLS/2022.2/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.3.2/bin:/tools/Xilinx/DocNav:/home/rosario/.local/bin:/home/rosario/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/var/lib/snapd/snap/bin)

20:38:51 ERROR : Timed out. 180 seconds have elapsed while waiting for XSCT server to launch.
20:38:51 ERROR : Failed to openhw "/tmp/hwspec_mb_design_wrapper9216242644888787270/mb_design_wrapper.xsa"
Reason: xsct server communication channel is not established.
20:38:51 INFO  : Launching XSCT server: xsct -n  -interactive /home/rosario/Documents/CR/cr-hardware-acceleration/temp_xsdb_launch_script.tcl
20:38:51 ERROR : (XSDB Server)which: no xlsclients in (/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_be/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.21.4/bin::/tools/Xilinx/Vivado/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vivado/2022.2/ids_lite/ISE/bin/lin64:/tools/Xilinx/Vitis/2022.2/aietools/bin:/tools/Xilinx/Vivado/2022.2/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib/python3.8/site-packages::/tools/Xilinx/Vivado/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vitis_HLS/2022.2/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.3.2/bin:/tools/Xilinx/DocNav:/home/rosario/.local/bin:/home/rosario/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/var/lib/snapd/snap/bin)

20:41:51 ERROR : Timed out. 180 seconds have elapsed while waiting for XSCT server to launch.
20:41:51 ERROR : Failed to openhw "/tmp/hwspec_mb_design_wrapper9216242644888787270/mb_design_wrapper.xsa"
Reason: xsct server communication channel is not established.
21:18:21 DEBUG : Logs will be stored at '/home/rosario/Documents/CR/cr-hardware-acceleration/IDE.log'.
21:18:23 INFO  : Platform repository initialization has completed.
21:18:23 INFO  : Launching XSCT server: xsct -n  -interactive /home/rosario/Documents/CR/cr-hardware-acceleration/temp_xsdb_launch_script.tcl
21:18:23 ERROR : (XSDB Server)which: no xlsclients in (/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_be/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.21.4/bin::/tools/Xilinx/Vivado/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vivado/2022.2/ids_lite/ISE/bin/lin64:/tools/Xilinx/Vitis/2022.2/aietools/bin:/tools/Xilinx/Vivado/2022.2/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib/python3.8/site-packages::/tools/Xilinx/Vivado/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vitis_HLS/2022.2/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.3.2/bin:/tools/Xilinx/DocNav:/home/rosario/.local/bin:/home/rosario/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/var/lib/snapd/snap/bin)

21:18:23 INFO  : Registering command handlers for Vitis TCF services
21:21:23 ERROR : Timed out. 180 seconds have elapsed while waiting for XSCT server to launch.
21:21:23 INFO  : Launching XSCT server: xsct -n  -interactive /home/rosario/Documents/CR/cr-hardware-acceleration/temp_xsdb_launch_script.tcl
21:21:23 ERROR : Failed to call init_repo
Reason: xsct server communication channel is not established.
21:21:23 ERROR : Exception raised in executing the command =Failed to call init_repo
Reason: xsct server communication channel is not established. 
21:21:23 ERROR : (XSDB Server)which: no xlsclients in (/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_be/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.21.4/bin::/tools/Xilinx/Vivado/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vivado/2022.2/ids_lite/ISE/bin/lin64:/tools/Xilinx/Vitis/2022.2/aietools/bin:/tools/Xilinx/Vivado/2022.2/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib/python3.8/site-packages::/tools/Xilinx/Vivado/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vitis_HLS/2022.2/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.3.2/bin:/tools/Xilinx/DocNav:/home/rosario/.local/bin:/home/rosario/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/var/lib/snapd/snap/bin)

21:24:23 ERROR : Timed out. 180 seconds have elapsed while waiting for XSCT server to launch.
21:24:23 ERROR : Failed to openhw "/tmp/hwspec_mb_design_wrapper13237659734920143631/mb_design_wrapper1.xsa"
Reason: xsct server communication channel is not established.
21:24:23 INFO  : Launching XSCT server: xsct -n  -interactive /home/rosario/Documents/CR/cr-hardware-acceleration/temp_xsdb_launch_script.tcl
21:24:23 ERROR : (XSDB Server)which: no xlsclients in (/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_be/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.21.4/bin::/tools/Xilinx/Vivado/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vivado/2022.2/ids_lite/ISE/bin/lin64:/tools/Xilinx/Vitis/2022.2/aietools/bin:/tools/Xilinx/Vivado/2022.2/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib/python3.8/site-packages::/tools/Xilinx/Vivado/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vitis_HLS/2022.2/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.3.2/bin:/tools/Xilinx/DocNav:/home/rosario/.local/bin:/home/rosario/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/var/lib/snapd/snap/bin)

21:27:23 ERROR : Timed out. 180 seconds have elapsed while waiting for XSCT server to launch.
21:27:23 ERROR : xsct server communication channel is not established.
21:27:23 INFO  : Launching XSCT server: xsct -n  -interactive /home/rosario/Documents/CR/cr-hardware-acceleration/temp_xsdb_launch_script.tcl
21:27:23 ERROR : (XSDB Server)which: no xlsclients in (/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_be/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.21.4/bin::/tools/Xilinx/Vivado/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vivado/2022.2/ids_lite/ISE/bin/lin64:/tools/Xilinx/Vitis/2022.2/aietools/bin:/tools/Xilinx/Vivado/2022.2/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib/python3.8/site-packages::/tools/Xilinx/Vivado/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vitis_HLS/2022.2/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.3.2/bin:/tools/Xilinx/DocNav:/home/rosario/.local/bin:/home/rosario/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/var/lib/snapd/snap/bin)

21:30:23 ERROR : Timed out. 180 seconds have elapsed while waiting for XSCT server to launch.
21:30:23 INFO  : Launching XSCT server: xsct -n  -interactive /home/rosario/Documents/CR/cr-hardware-acceleration/temp_xsdb_launch_script.tcl
21:30:23 ERROR : (XSDB Server)which: no xlsclients in (/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_be/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.21.4/bin::/tools/Xilinx/Vivado/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vivado/2022.2/ids_lite/ISE/bin/lin64:/tools/Xilinx/Vitis/2022.2/aietools/bin:/tools/Xilinx/Vivado/2022.2/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib/python3.8/site-packages::/tools/Xilinx/Vivado/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vitis_HLS/2022.2/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.3.2/bin:/tools/Xilinx/DocNav:/home/rosario/.local/bin:/home/rosario/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/var/lib/snapd/snap/bin)

21:33:23 ERROR : Timed out. 180 seconds have elapsed while waiting for XSCT server to launch.
21:33:23 INFO  : Launching XSCT server: xsct -n  -interactive /home/rosario/Documents/CR/cr-hardware-acceleration/temp_xsdb_launch_script.tcl
21:33:23 ERROR : Failed to openhw "/tmp/hwspec_mb_design_wrapper13237659734920143631/mb_design_wrapper1.xsa"
Reason: xsct server communication channel is not established.
21:33:23 ERROR : (XSDB Server)which: no xlsclients in (/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_be/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.21.4/bin::/tools/Xilinx/Vivado/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vivado/2022.2/ids_lite/ISE/bin/lin64:/tools/Xilinx/Vitis/2022.2/aietools/bin:/tools/Xilinx/Vivado/2022.2/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib/python3.8/site-packages::/tools/Xilinx/Vivado/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vitis_HLS/2022.2/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.3.2/bin:/tools/Xilinx/DocNav:/home/rosario/.local/bin:/home/rosario/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/var/lib/snapd/snap/bin)

21:36:23 ERROR : Timed out. 180 seconds have elapsed while waiting for XSCT server to launch.
21:36:23 ERROR : Failed to query RDI_DATADIR
Reason: xsct server communication channel is not established.
21:36:23 INFO  : Launching XSCT server: xsct -n  -interactive /home/rosario/Documents/CR/cr-hardware-acceleration/temp_xsdb_launch_script.tcl
21:36:23 ERROR : Exception raised in executing the command =Failed to query RDI_DATADIR
Reason: xsct server communication channel is not established. 
21:36:23 ERROR : (XSDB Server)which: no xlsclients in (/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_be/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.21.4/bin::/tools/Xilinx/Vivado/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vivado/2022.2/ids_lite/ISE/bin/lin64:/tools/Xilinx/Vitis/2022.2/aietools/bin:/tools/Xilinx/Vivado/2022.2/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/bin:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib:/tools/Xilinx/Vivado/2022.2/tps/lnx64/python-3.8.3/lib/python3.8/site-packages::/tools/Xilinx/Vivado/2022.2/tps/lnx64/jre11.0.11_9/bin:/tools/Xilinx/Vitis_HLS/2022.2/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/arm/lin/bin:/tools/Xilinx/Vitis/2022.2/gnu/microblaze/linux_toolchain/lin64_le/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-linux-gnueabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch32/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-linux/bin:/tools/Xilinx/Vitis/2022.2/gnu/aarch64/lin/aarch64-none/bin:/tools/Xilinx/Vitis/2022.2/gnu/armr5/lin/gcc-arm-none-eabi/bin:/tools/Xilinx/Vitis/2022.2/tps/lnx64/cmake-3.3.2/bin:/tools/Xilinx/DocNav:/home/rosario/.local/bin:/home/rosario/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/var/lib/snapd/snap/bin)

21:39:23 ERROR : Timed out. 180 seconds have elapsed while waiting for XSCT server to launch.
21:39:23 ERROR : Failed to openhw "/tmp/hwspec_mb_design_wrapper13237659734920143631/mb_design_wrapper1.xsa"
Reason: xsct server communication channel is not established.
